c_src/enif_protobuf.c

```diff
@@ -20,6 +20,18 @@ make_atom(ErlNifEnv *env, const char *name)
     return enif_make_atom(env, name);
 }
 
+
+char*
+get_atom(ErlNifEnv *env, ERL_NIF_TERM term, char *buf, unsigned size)
+{
+    if (enif_get_atom(env, term, buf, size, ERL_NIF_LATIN1)) {
```

Comment:
question: why cann't the caller call enif_get_atom(,,,, ERL_NIF_LATIN1)?

Reply:
no reason actually..
caller also can call.. i felt it is in-line with the style of the repo..
would you like the other way better?

Reply:
what you have is fine.